### PR TITLE
Initial addition of files

### PR DESCRIPTION
**Pull request by MikeCamel**
Completed initial set of file additions.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -82,31 +82,32 @@ mod models {
...
-                                        );
-                                        keeploaderlist.push(keeploader);
-                                        println!("Pushed keeploader to list");
-                                    }
-                                    Err(e) => println!("not a useful reply {}", e),
-                                }
+                            let mut iterator = deserializer.into_iter::<serde_json::Value>();
+                            //we're only expecting one command
+                            let json_pair = iterator.next();
+                            //for json_pair in iterator {
```
Comment (lkatalin) [inline src/main.rs]: Probably want to get rid of this comment.

File: src/main.rs [hunk 2]
```diff
@@ -82,31 +82,32 @@ mod models {
...
+                                        "keeploader on {} has kuuid {}",
+                                        path.display(),
+                                        keeploader.kuuid,
+                                    );
+                                    keeploaderlist.push(keeploader);
+                                    println!("Pushed keeploader to list");
+                                }
+                                Err(e) => println!("not a useful reply {}", e),
                             }
+                            //  break;
```
Comment (lkatalin) [inline src/main.rs]: And this comment.

File: src/keep-manager-tester.rs [hunk 3]
```diff
@@ -16,7 +16,7 @@ fn main() {
     command_list_all.insert("command".to_string(), "list-all".to_string());
     let mut command_new_keep: HashMap<String, String> = HashMap::new();
     command_new_keep.insert("command".to_string(), "new-keep".to_string());
-    command_new_keep.insert("keep-arch".to_string(), "wasi".to_string());
+    command_new_keep.insert("keep-arch".to_string(), KEEP_ARCH_WASI.to_string());
```
Comment (lkatalin) [inline src/keep-manager-tester.rs]: Where is `KEEP_ARCH_WASI` defined?

File: src/main.rs [hunk 4]
```diff
@@ -147,10 +147,9 @@ mod filters {
         //TODO - the chances should be _pretty_ low, but should
         // we check for an existing keep?
         //TODO - remove hard-coded systemd-escape sequence ("\x20")
-        let service_cmd = format!(
-            "enarx-keep@{}\\x20{}\\x20{}.service",
-            new_kuuid, apploaderbindaddr, apploaderbindport
-        );
+        let operation_type = "daemon";
+        let service_cmd = format!("enarx-keep@{}\\x20{}.service", operation_type, new_kuuid);
+        //let service_cmd = format!("enarx-keep@\"{}\\x20{}\".service", operation_type, new_kuuid);
```
Comment (lkatalin) [inline src/main.rs]: Also an extra comment here.

**Review by lkatalin — COMMENTED** [review]
The small changes look reasonable compared to what was there before. However, there were some extra comments left in the code that seem like they should be removed.

Also, what are the "file additions" referred to in the PR description?

File: src/main.rs (hunk 4 above)
Comment (MikeCamel) [reply src/main.rs]: This code has gone.

File: src/keep-manager-tester.rs (hunk 3 above)
Comment (MikeCamel) [reply src/keep-manager-tester.rs]: This code has gone (though the answer is "lib.rs"

File: src/main.rs (hunk 1 above)
Comment (MikeCamel) [reply src/main.rs]: this code is gone.

File: src/main.rs (hunk 2 above)
Comment (MikeCamel) [reply src/main.rs]: this code is gone.

**Comment by MikeCamel** [comment]
@npmccallum - I'd appreciate your input on the clippy fail, please.  Not sure if the code needs fixing (not convinced), or if we need to change the clippy settings.

**Pull request closed without merging**